sceptre/strides/config/prod/github-oidc-sage-bionetworks-it.yaml

```diff
@@ -1,14 +1,17 @@
 template:
   type: http
-  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.1/templates/IAM/github-oidc-provider.j2
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
 stack_name: github-oidc-sage-bionetworks-it
+Dependencies:
+  - prod/github-oidc-provider.yaml
 parameters:
+  ProviderRoleName: github-oidc-sage-bionetworks-it
+  ProviderArn: !stack_output_external github-oidc-provider::ProviderArn
```

Comment:
you'll need to create a follow on PR to update GH action role-to-assume for sceptre-strides, https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/.github/workflows/main.yaml#L130-L136